service/service.hpp

```diff
@@ -174,7 +168,7 @@ namespace raven
         const config_create_answer answer{"Foo", "Foo"};
         json::json response_json_data;
         to_json(response_json_data, answer);
-        return response_json_data;
+        sock.write(response_json_data.dump().data(), static_cast<unsigned int>(response_json_data.dump().size()));
```

Comment:
I don't know the json API to be sûre, but this looks suspicious.
Does dump return a temporary ? Because in that case I think this code isn't correct (the data pointer may be pointing to potentially dead memory)

Reply:
It's return a string by copy

Reply:
Ok, I believe we're in undefined behaviour territory in this case.